helper/tlsutil/generate.go

```diff
@@ -116,17 +124,53 @@ func GenerateCA(opts CAOpts) (string, string, error) {
 		days = 365
 	}
 
+	country := opts.Country
+	if country == "" {
+		country = ("US")
+	}
+
+	postalCode := opts.PostalCode
+	if postalCode == "" {
+		postalCode = ("94105")
+	}
+
+	province := opts.Province
+	if province == "" {
+		province = ("CA")
+	}
+
+	locality := opts.Locality
+	if locality == "" {
+		locality = ("San Francisco")
+	}
+
+	streetAddress := opts.StreetAddress
+	if streetAddress == "" {
+		streetAddress = ("101 Second Street")
+	}
+
+	organization := opts.Organization
+	if organization == "" {
+		organization = ("HashiCorp Inc.")
+	}
+
+	organizationalUnit := opts.OrganizationalUnit
+	if organizationalUnit == "" {
+		organizationalUnit = ("Nomad")
+	}
```

Comment:
The assignments do not require parentheses; so `organizationalUnit = ("Nomad")` can become `organizationalUnit = "Nomad"` for example.